css/index-flex1.css

```diff
@@ -163,70 +172,101 @@ a {
     
    
 }
+.boxes1-5 {
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+height: 100px;
+width: 100%;
+}
+
+
+
+.boxes6-10 {
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+height: 100px;
+width: 100%;
+}
+
 div.boxes {
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
     margin: 30px;
+    max-width: -webkit-fill-available;
 }
 
 .for-spacing {
     width: 400px;
 }
 .box1 {
-   
+padding: 45px;   
 background-color: teal;
 width: 12.5%;
 margin-right: 10px;
 
 } 
  .box2 {
-    
+    padding: 45px;     
 background-color: gold;
 width: 12.5%;
 margin-right: 10px;
  } 
 
  .box3{
-   
+    padding: 45px;   
     background-color: cadetblue;
     width: 12.5%;
     margin-right: 10px;
  }
  .box4{
+    padding: 45px;   
     background-color: coral;
     width: 12.5%;
     margin-right: 10px;
  }
  .box5{
+    padding: 45px;   
     background-color: crimson;
     width: 12.5%;
     margin-right: 10px;
  }
```

Comment:
There is a way to do this so you don't have to indicate two separate classes for each row. When you display flex on the container that the boxes are in, it opens up so many new properties that can be used. One is flex-wrap. I would investigate that for this instance.